### PR TITLE
Add archey

### PR DESCRIPTION
**Pull request by oxguy3**
Archey is a tool that displays some basic system information alongside an ASCII version of your OS's logo. The tool exists on both OS X and Linux, but they're separate codebases that work slightly differently, so they get separate pages.

The OS X version has a couple of flags for tweaking the output, while the Linux version has absolutely no options (and I would have never bothered adding it if I wasn't also simultaneously adding the OS X one). I used the long forms of the flags for the OS X page, even though every flag does have a shortcut, simply because I don't think anyone uses archey frequently enough to memorize the flags.

### CODE REVIEW

**Comment by rubenvereecken** [comment]
It's good you think of intuitiveness of the flags. Good call.

:+1:

File: pages/osx/archey.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# archey
+
+> Simple tool for stylishly displaying system information.
+
+- Show system information in stylish output:
```
Comment (rubenvereecken) [inline pages/osx/archey.md]: You imply the linux archey is not very stylish :P?

Comment (oxguy3) [inline pages/osx/archey.md]: Hehe, no, it'd probably make sense to remove that "in stylish output" bit from the OS X one too.

**Comment by rubenvereecken** [comment]
Ah, you updated. Great stuff

**Pull request merged**